cross/libopus/Makefile

```diff
@@ -0,0 +1,18 @@
+PKG_NAME = libopus
```

Comment:
Rename to `opus` (including dirname), and go from there? I don't see a reference to `libopus` anywhere else (well, the PLIST, but that doesn't matter here)